tyr-core/src/test/java/org/jboss/tyr/config/EmptyAdditionalResourcesPropertyTest.java

```diff
@@ -0,0 +1,43 @@
+package org.jboss.tyr.config;
+
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.QuarkusTestProfile;
+import io.quarkus.test.junit.TestProfile;
+import org.jboss.tyr.InvalidPayloadException;
+import org.jboss.tyr.TestUtils;
+import org.jboss.tyr.additional.resource.DummyAdditionalCommand;
+import org.jboss.tyr.check.TemplateChecker;
+import org.jboss.tyr.whitelist.WhitelistProcessing;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import javax.inject.Inject;
+import java.util.Map;
+
+@QuarkusTest
+@TestProfile(EmptyAdditionalResourcesPropertyTest.Profile.class)
+public class EmptyAdditionalResourcesPropertyTest {
+
+    @Inject
+    WhitelistProcessing whitelistProcessing;
+
+    @Inject
+    TemplateChecker templateChecker;
+
+    public static class Profile implements QuarkusTestProfile {
+        @Override
+        public Map<String, String> getConfigOverrides() {
+            return Map.of("tyr.additional-resources", "");
+        }
+    }
+
+    @Test
+    public void emptyAdditionalResourcesPropertyTest() throws InvalidPayloadException {
+        String result = templateChecker.checkPR(TestUtils.TEST_PAYLOAD);
+        whitelistProcessing.processPRComment(TestUtils.ISSUE_PAYLOAD);
+
+        Assertions.assertTrue(result.isEmpty());
+        Assertions.assertFalse(DummyAdditionalCommand.isTriggered());
```

Comment:
It's always good idea to provide some failure message, can you provide some please?